schemas/app-sre/automated-action-1.yml

```diff
@@ -0,0 +1,27 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /app-sre/automated-action-1.yml
+  labels:
+    "$ref": "/common-1.json#/definitions/labels"
+  operationId:
+    type: string
+    description: The automated action OpenApi operation id.
+  description:
+    type: string
+  retries:
+    type: number
+    description: "Number of times to retry the action. Default: 5"
+  maxOps:
```

Comment:
It is important to force that every automated action has a limit. So that one forgets to assign it.

Reply:
I've changed the description to 6 ops/hour. This should be a good sensitive default value

Reply:
Done in b5a0e84